js/product-details.js

```diff
@@ -0,0 +1,255 @@
+          textBig.classList.add("details-notes--margin");
+          valueConteinerBig.appendChild(textBig);
+        }
+      } else {
+        let feedbackSmoll = document.createElement("div");
+        feedbackSmoll.innerHTML = drinkKey[key];
+        feedbackSmoll.classList.add("details-notes--margin");
+        valueConteinerSmoll.appendChild(feedbackSmoll);
+
+        let feedbackBig = document.createElement("div");
+        feedbackBig.innerHTML = drinkKey[key];
+        feedbackBig.classList.add("details-notes--margin");
+        valueConteinerBig.appendChild(feedbackBig);
+      }
+    }
+    conteinerSmoll.appendChild(valueConteinerSmoll);
+    conteinerBig.appendChild(valueConteinerBig);
+  } else if (Object.prototype.toString.call(drink[key]) == "[object Array]") {
+    let valueConteinerSmoll = document.createElement("div");
+    valueConteinerSmoll.classList.add("product-details-value--smoll");
+    valueConteinerSmoll.classList.add("product-details-value--smoll--none");
+    valueConteinerSmoll.classList.add("product-details-value--smoll--" + key);
+    let valueConteinerBig = document.createElement("div");
+    valueConteinerBig.classList.add("product-details-value--big");
+    valueConteinerBig.classList.add("product-details-value--big--none");
+    valueConteinerBig.classList.add("product-details-value--big--" + key);
+    for (i = 0; i < drink[key].length; i++) {
+      let textSmoll = document.createElement("div");
+      textSmoll.innerHTML = drink[key][i];
+      textSmoll.classList.add("details-notes--margin");
+      valueConteinerSmoll.appendChild(textSmoll);
+      let textBig = document.createElement("div");
+      textBig.innerHTML = drink[key][i];
+      textBig.classList.add("details-notes--margin");
+      valueConteinerBig.appendChild(textBig);
+    }
+    conteinerSmoll.appendChild(valueConteinerSmoll);
+    conteinerBig.appendChild(valueConteinerBig);
+  } else {
+    let valueConteinerSmoll = document.createElement("div");
+    valueConteinerSmoll.innerHTML = drink[key];
+    valueConteinerSmoll.classList.add("product-details-value--smoll");
+    valueConteinerSmoll.classList.add("product-details-value--smoll--none");
+    valueConteinerSmoll.classList.add("product-details-value--smoll--" + key);
+    conteinerSmoll.appendChild(valueConteinerSmoll);
+    let valueConteinerBig = document.createElement("div");
+    valueConteinerBig.innerHTML = drink[key];
+    valueConteinerBig.classList.add("product-details-value--big");
+    valueConteinerBig.classList.add("product-details-value--big--none");
+    valueConteinerBig.classList.add("product-details-value--big--" + key);
+    conteinerBig.appendChild(valueConteinerBig);
+  }
+};
+
+const clickButton = function() {
+  const productDetails = document.getElementsByClassName("details-button");
+  Array.from(productDetails).forEach(function(item) {
+    item.addEventListener("click", function() {
+      if (windowWidth() > 769) {
```

Comment:
numbers should be moved to constants, because I don't understand what 768 means.
if it's sreen size in which we should see articles in a row - call the variable SIZE_FOR_ARTICLES_IN_ROW
and so on
constants names that are not dynamically created should be named as I wrote below - caps lock and underscores